source/usage-examples/findOne.go

```diff
@@ -0,0 +1,54 @@
+//Package main will provides a code example showing
+//how to find a document using the MongoDB Go Driver.
+package main
+
+import (
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+)
+
+//main finds and prints the lowest rated movie titled "The Room" in the sample dataset.
+func main() {
+
+	// Replace the uri string with your MongoDB deployment's connection string.
+	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority"
```

Comment:
This is a topic that I know you want to discuss from your recent comment, so maybe we can figure out the best way to proceed here.

Reply:
Thanks for the note. I have added the connection guide note. Going forward, I will try to be more mindful of requesting things of readers without presenting appropriate background information. 

I think a good solution might be to have a working connection string to Free-Tier Atlas Cluster controlled by the DevEd-Drivers team with a Read Only user for testing code snippets. So, we would put a valid connection string for a Read Only user in code snippets. I think this would be great from a reader perspective as no configuration would be required to run Read-Only code snippets. Thoughts?

This would require a discussion with the team, and we would have to have a conversation about potential security/billing implications.

Reply:
I made a [google doc](https://docs.google.com/document/d/1y_HYv8wjVBTjPXmL8IjaXGB4jUQirI9vJYOw_Wu0Uew/edit) and shared it with the team to think about creating alternatives for placeholder connection URIs in docs. Have added this to my trello as well, but need to get better at working on trello tasks regularly.